shim/toolchain.bzl

```diff
@@ -68,7 +68,7 @@ def _cxx_toolchain(ctx):
                 nm = RunInfo(args = ["nm"]),
                 # objcopy = ctx.attrs.objcopy_for_shared_library_interface[RunInfo],
                 # ranlib = ctx.attrs.ranlib[RunInfo],
-                ranlib = RunInfo(args = ["raninfo"]),
+                ranlib = RunInfo(args = ["ranlib"]),
```

Comment:
We no longer use shim/toolchain, since we now use the system toolchains in the examples